contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--2.4.0--3.0.0.sql

```diff
@@ -0,0 +1,29 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
```

Comment:
With accordance to https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1045#discussion_r1072608467
In case there is scenario where one upgrades first from 14.6->14.7 and then 14.7->15.2. 
We would need the collation changes that were also done for 2.3->3.0 . Both of the upgrade script should be identical

Reply:
Yes, this changes are required to make the MVU work.